components/dashboard/src/Login.tsx

```diff
@@ -284,8 +280,13 @@ const LoginContent = ({
                         <Heading2>Open a cloud development environment</Heading2>
                         <Subheading>for the repository {repoPathname?.slice(1)}</Subheading>
                     </>
-                ) : (
+                ) : !isGitpodIoUser ? (
                     <Heading1>Log in to Gitpod</Heading1>
+                ) : (
+                    <>
+                        <Heading1>Log in to Gitpod Classic</Heading1>
+                        <Subheading>Hosted by us</Subheading>
```

Comment:
copy nit: "Hosted by us" could theoretically also be used for Gitpod Flex. Is that intentional?

Reply:
> could theoretically also be used for Gitpod Flex

I don't think so 🤔 

As of now, Gitpod Flex is not "hosted" by us, it's hosted in customer's runners

Reply:
The point is to highligh that workspaces are hosted by us, one does not need to self host or run on local.

I do agree it looks out of the context though. Not sure how could be rephrased better.

Reply:
Yes, the runners are hosted by users, but the app people use as the primary interface is hosted by us either way. It's not clear what is "Hosted by us" exactly. This is not that big of a deal though compared to my other comment, I think we can resolve.